image.php

```diff
@@ -51,7 +51,7 @@
 
 // FIXME: Legacy stuff - should be removed after a version or so
 if (!filter_has_var(INPUT_GET, 'object_type')) {
-    $_GET['object_type'] = 'album';
```

Comment:
what does this change do? i'm assuming it's setting a default type for art?

Reply:
To be honest I don't remember. I changed that line 1 1/2 years ago in a lot of debugging and try and error sessions. When I review that now I would agree. I think this default setting should not be hard coded, but depending on the config file.